diff a/qtbase/configure.cmake b/qtbase/configure.cmake
index a5d968d..a665c49 100644
--- a/qtbase/configure.cmake
+++ b/qtbase/configure.cmake
@@ -1525,11 +1525,6 @@ qt_configure_add_report_entry(
     MESSAGE "Some tests might fail to build when targeting WASM without -feature-thread."
     CONDITION WASM AND QT_BUILD_TESTS AND NOT QT_FEATURE_thread
 )
-qt_configure_add_report_entry(
-    TYPE ERROR
-    MESSAGE "Building Qt with C++20 is not supported with MSVC 2019."
-    CONDITION QT_FEATURE_cxx20 AND MSVC AND MSVC_VERSION LESS "1930"
-)
 qt_configure_add_report_entry(
     TYPE ERROR
     MESSAGE "You cannot force both system and bundled libraries."
